aten/src/ATen/native/native_functions.yaml

```diff
@@ -269,6 +269,7 @@
   variants: method
 
 - func: absolute.out(Tensor self, *, Tensor(a!) out) -> Tensor(a!)
+  use_c10_dispatcher: hacky_wrapper_for_legacy_signatures
```

Comment:
Why wouldn't all these single-return ops go in the preceding PR?

Reply:
Ah, reread the PR description. It would be nice to move these to the previous PR for hygiene, I think - shouldn't be too hard, right?